source/applySpec.js

```diff
@@ -1,12 +1,19 @@
 import _curry1 from './internal/_curry1';
 import apply from './apply';
 import curryN from './curryN';
-import map from './map';
 import max from './max';
 import pluck from './pluck';
 import reduce from './reduce';
 import values from './values';
+import fromPairs from './fromPairs';
+import toPairs from './toPairs';
 
+// Use the built-in map function to avoid issues with specs what include a "map" key
+function mapValues(fn, obj) {
```

Comment:
Instead of writing our own function here, can you use the same process `map` does for mapping Objects: https://github.com/ramda/ramda/blob/master/source/map.js#L52?

Reply:
The only problem with that is that it depends upon `_reduce`, which adds transducing behaviors that I don't think we want here.

Reply:
Happy to do something along the lines of 
```
function mapValues(fn, obj) {
  return _reduce(function(acc, key) {
    acc[key] = fn(obj[key]);
    return acc;
  }, {}, keys(obj));
}
```

Unless there's some transducing behavior here that we don't want?

Reply:
Ah, comment below works as well